verilog/parser/verilog_lexer_unittest.cc

```diff
@@ -79,13 +79,16 @@ static std::initializer_list<LexerTestData> kCommentTests = {
 // treating attributes lists as C-style comments,
 // except they are not returned as comment blocks.
 static std::initializer_list<SimpleTestData> kAttributeTests = {
-    {"(**)"},        {"(*     *)"},
```

Comment:
which of the original tests don't work anymore and why ?

We shoud be careful with removing any tests mentioned here as they probably come from a real-world example. Err on the side of adding tests but never remove them unless the original test was clearly faulty.

Reply:
`(* x)`, `(* **  *)`, `(***)`, `(** **)` - these tests have been removed. I am not sure about their correctness because I have not found such cases anywhere. The systemverilog manual also does not mention such cases.

Reply:
I found some (small) regression in one of our internal tests. Will construct a test-case out of it tomorrow.

Reply:
Alright, in #1695 I added an example.

Reply:
Thanks. Will look into that.